Sources/StreamChat/Database/DTOs/ChannelDTO.swift

```diff
@@ -20,6 +20,10 @@ class ChannelDTO: NSManagedObject {
     @NSManaged var updatedAt: Date
     @NSManaged var lastMessageAt: Date?
 
+    // The oldest message in the current batch of messages. This property only lives locally, and it is useful
+    // to filter out older pinned messages that do not belong to the current batch of messages in the channel.
+    @NSManaged var oldestMessageAt: Date
+
```

Comment:
`// The oldest message in the current batch of messages.` is not correct, it's the oldest message of the channel we have locally coming from a regular channel query.

Reply:
Makes sense, thanks 👍